ipatests/test_integration/test_ipahealthcheck.py

```diff
@@ -143,6 +143,31 @@ def test_run_ipahealthcheck_list_source(self):
         for source in sources:
             assert source in result.stdout_text
 
+    def test_human_output(self):
+        """
+        Test that in human output the severity value is correct
+
+        Only the SUCCESS (0) value was being translated, otherwise
+        the numeric value was being shown (BZ 1752849)
+        """
+        self.master.run_command(["systemctl", "stop", "sssd"])
```

Comment:
@rcritten.
May be too late... sorry.
If this test fails, then the next ones which rely on running `sssd` will fail too.
I suggest using the pytest's `fixture` here, something like:

```python
import pytest

@pytest.fixture
def sssd_stop():
    print('sssd stop')
    yield
    print('sssd start')

class TestMy:
    def test_1(self, sssd_stop):
        print('test_1')
```

Reply:
That's a good idea, I'll look into it.

Reply:
I need to pass in the master value in order to execute the stop/start. I could do that via a context but it seems like it would be clearer if I just moved the restart up so that it executes right after ipa-healthcheck which can fail but won't raise an exception so sssd should always be restarted.

Reply:
alternatively, a fixture may be defined as a method:
```python
class TestMy:
    master='Master'

    @pytest.fixture
    def sssd_stop(self):
        print('sssd stop on ', self.master)
        yield
        print('sssd start on ', self.master)

    def test_1(self, sssd_stop):
        print('test_1')
```
But I'm happy with the current version.

Reply:
@rcritten, the exception could be other than `CalledProcessError`. That's why the fixture is preferred.

Reply:
Yeah, I guess so, but it is just calling exec so I'd find it extremely unlikely to be anything else.
And anything that might return something else would be so catastrophically bad that not being able to restart sssd for this test case would be the least of our issues.

Reply:
You could also wrap the call into ``try: run_healthcheck(); finally: start sssd`` block.

Reply:
added try/except/else/finally

Reply:
as @tiran proposed it's enough `try->finally` (without except/else).